vignettes/estimate_ascertainment.Rmd

```diff
@@ -261,20 +229,9 @@ df_covid <- get_national_data(
 df_covid_subset <- filter(df_covid, date <= "2020-05-31")
 ```
 
-Finally, we put the ascertainment estimates for all countries with large 
-outbreaks in a single table and summarise the results in a single figure. We 
-define a large outbreak as one which has caused at least 100,000 deaths. We 
-focus on these countries as the resulting table and figure become unwieldy if
-they are made any larger. We focus on the period between the start of each 
-outbreak to the 1st June 2020. We do so as it was this period where 
-under-ascertainment of cases and infections was likely the highest, as tests
-were still being developed and had not been made widely available in many 
-countries.
-
-To calculate all ascertainment estimates for all countries with
-large COVID-19 outbreaks up to the 1st June 2020, we use the following command —
-this codeblock depends on the package `dplyr`, for ease with manipulating
-large data.frames which are structured by group:
+We adopt a data science approach to apply the `estimate_reporting()` function across data grouped by country.
```

Comment:
I'm not fully understanding what a data science approach means here, is this referring to the fact that the users will have to format the data themselves using functions from `dplyr`?

Reply:
Yes - now changed to make that clearer.

> We adopt data science tools to iteratively apply the `estimate_reporting()` function across data grouped by country.